source/pluginmanager/PluginManager.py

```diff
@@ -51,7 +51,12 @@ def get_plugins_of(self, organization):
     def get_required_info_of(self, organization, plugin_name):
         """ Get the required information of a specific """
         plugin = self.__get_plugin(organization, plugin_name)
-        return plugin["required_info"]
+        required_info = plugin["required_info"]
+        required_info['name'] = 'default_name'
+        return {'organization': organization
+                , 'plugin_name': plugin_name
+                , 'required_info': plugin["required_info"]
+                }
 

```

Comment:
Just as a suggestion, maybe it is neater to perform these mutations on the required info in the logic module calling the plugin manager.